server/src/repositories/album.repository.ts

```diff
@@ -62,9 +62,9 @@ export class AlbumRepository implements IAlbumRepository {
     return this.repository.find({
       where: [
         { ownerId, assets: { id: assetId } },
-        { sharedUsers: { id: ownerId }, assets: { id: assetId } },
+        { sharedUsers: { user: Equal(ownerId) }, assets: { id: assetId } },
```

Comment:
fix this one please. it should be { user: { id: ownerId } } I think. Or potentially { userId: ownerId } since that column is on the album user entity now.

Reply:
Thanks, I missed these